lib/src/extensions/v_stream.dart

```diff
@@ -0,0 +1,71 @@
+import 'dart:async';
+
+import 'package:rxdart/rxdart.dart';
+
+class VStream {
+  /// Similar to `Stream.periodic`, this
+  /// creates a stream that emits when it is being listened to and then repeatedly firest at [period] intervals.
+  ///
+  /// The event values are computed by invoking [computation].
+  /// The argument to this callback is an integer that starts with 0 (the initial event)
+  /// and is incremented for every periodic event.
+  static Stream<T?> initialPeriodic<T>(Duration period, [T Function(int)? computation]) {
+    T? _computation(int index) => computation == null ? null : computation(index);
+
+    return Stream<T?>.periodic(
+      period,
+      (computationCount) => _computation(computationCount + 1),
+    ).shareValueSeeded(_computation(0));
+  }
+
+  /// Generates a stream that emits the current [DateTime] every minute.
+  ///
+  /// The stream emits the current [DateTime] every minute, with the option to emit the initial tick immediately after being listened to.
+  /// The time of the ticks can be offset by a specified duration.
+  ///
+  /// The function throws an [ArgumentError] if the `tickOffset` is more than 59 seconds.
+  static Stream<DateTime> minutely({
+    bool initialTick = true,
+    Duration tickOffset = Duration.zero,
+  }) {
+    if (tickOffset > const Duration(seconds: 59)) {
```

Comment:
Improvement:
Could be an `assert`

Reply:
I rather throw here, because in this case it does not make sense to have a delay that overflows a minute:
1. I did not want to bother handling any potential issue this could cause. (the calculation logic inside the Timer.period) .. this is mainly chatgpt generated and it works well with <1 min delay scenario, I did not want to mess with >1 min cases ... maybe it would work, but throwing is safer :D